<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(1, 38, 38);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(1, 64, 58);
color: rgb(255, 255, 255);
font: 25 8pt &quot;Microsoft YaHei Light&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="row0col0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 140, 127);
font: 25pt &quot;MS PGothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row0col1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 140, 127);
font: 25pt &quot;MS PGothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row0col2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 140, 127);
font: 25pt &quot;MS PGothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row1col0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 140, 127);
font: 25pt &quot;MS PGothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row1col1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 140, 127);
font: 25pt &quot;MS PGothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row1col2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 140, 127);
font: 25pt &quot;MS PGothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row2col0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 140, 127);
font: 25pt &quot;MS PGothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row2col1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 140, 127);
font: 25pt &quot;MS PGothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="row2col2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 140, 127);
font: 25pt &quot;MS PGothic&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 140, 127);
font: 25 15pt &quot;Segoe UI Light&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
